src/Todo/Model.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
This extensions should be added to default extensions since it's completely safe. And it's convenient to not write it manually every time.